NavigationMenu.md

```diff
@@ -33,6 +33,12 @@
   * [Update space membership](./guides/V1_update_space.md)
 
 
+* ## [Reactions](./guides/V1_wwsg_Reactions.md)
```

Comment:
since this is experimental, it should not go in the NavigationMenu file, can you remove this for now @pjoviak